gold-cc-cvc-input.html

```diff
@@ -183,10 +184,27 @@
           this._handleAutoValidate();
         },
 
-        _computeRequiredLength: function(cardType) {
+        _onCardTypeChanged: function(cardType, oldCardType) {
+          // The initial property assignment is handled by `ready`.
+          if (oldCardType == undefined)
+            return;
+          if (this.value) this._handleAutoValidate();
+        },
+
+        _computeMaxLength: function(cardType) {
+          if (!cardType) return 4;
           return this._computeIsAmex(cardType) ? 4 : 3;
         },
 
+        _validateLength: function(value, cardType) {
```

Comment:
This can just be `validateLength: function()`, and it can use `this.value` and `this.cardType`. It's not an observer, so it doesn't need to have the values passed as a parameter